### DISCUSSION TITLE
DHCP server in a Podman container

### DISCUSSION THREAD

**Discussion by AndresIsaac5** [containers/podman#15274] [Q&A]
hello, please help, I need to know how to implement a DHCP server in a container using podman, I have used the networkboot / dhcp in docker. but I haven't found a way to do it with podman.

**Comment by rhatdan** [comment]
@flouthoc @mheon @Luap99 PTAL

**Comment by mheon** [comment]
It should work exactly as it does in Docker (assuming you're using root Podman - I don't think a DHCP server is possible in rootless). I see no differences in our network stacks that would require significant changes to a DHCP server container (I expect it'll be macvlan or `--net=host`, which both work in Podman).

**Comment by rhatdan** [comment]
So you want a dhcp server to run in rootless mode, and allow other systems to use it,  You could lower the minimal port for non root users to port 67.
